tdrs-backend/tdpservice/parsers/case_consistency_validator.py

```diff
@@ -171,22 +187,55 @@ def __validate_section2(self, num_errors):
         num_errors += self.__validate_t5_aabd_and_ssi()
         return num_errors
 
-    def __validate_family_affiliation(self, num_errors, t1s, t2s, t3s, error_msg):
+    def __validate_family_affiliation(self,
+                                      num_errors,
+                                      t1_model_name, t1s,
```

Comment:
maybe the logic for `tx_model_name` can be pulled into another function. memoization can help reduce re-computes for runs on the same case/file types

Reply:
@jtimpe take a look at what I did and let me know what you think.